3.0-RELEASE-NOTES.md

```diff
@@ -102,3 +102,8 @@ bulk `updateOrCreate`, you could use `async.each` or `Promise.all` to
 repeatedly call `updateOfCreate` for each instance.
 
 See [related code change](https://github.com/strongloop/loopback-datasource-juggler/pull/889) here.
+
++## Change default of `Model.settings.updateOnLoad` to true
```

Comment:
This is not correct description. AFAICT from the code change, the option is removed entirely.